core/commands/name/publish.go

```diff
@@ -83,19 +83,17 @@ Alternatively, publish an <ipfs-path> using a valid PeerID (as listed by
 		cmds.StringOption(ttlOptionName, "Time duration this record should be cached for. Uses the same syntax as the lifetime option. (caution: experimental)"),
 		cmds.StringOption(keyOptionName, "k", "Name of the key to be used or a valid PeerID, as listed by 'ipfs key list -l'.").WithDefault("self"),
 		cmds.BoolOption(quieterOptionName, "Q", "Write only final hash."),
+		cmds.BoolOption(v1compatOptionName, "Produce a backward-compatible IPNS Record by including fields for both V1 and V2 signatures.").WithDefault(true),
```

Comment:
:thought_balloon:  This PR should have  information about this + related IPIP (when we create it  in `/docs/changelogs/v0.22.md` (ok to create a placeholder TODO in markdown for now, we can fill it up before 0.22  release)